### PR TITLE
Add calOptions to createEventInteractively

### PR DESCRIPTION
**Pull request by syndbg**
calOptions are found in `Calendar.m`'s `createEventInteractively` but never passed.

This should do the trick.

### CODE REVIEW

**Comment by EddyVerbruggen** [comment]
Good find!

I don't like breaking current usage (the success handler has been replaced by the options object in your PR).

Can you think of a way to support both old a new usage? Two things come to mind:
- Detecting whether calOptions is a function or and object
- Creating a new function `createEventInteractivelyWithOptions

Curious to what you think.

**Comment by syndbg** [comment]
`Detecting whether calOptions is a function or and object` - will probably confuse the hell out of people new to the plugin.
`createEventInteractivelyWithOptions` - sounds like the best approach!

**Comment by EddyVerbruggen** [comment]
That's what I think as well. Can you update the PR perhaps?

**Comment by syndbg** [comment]
Will do after some hours. I'm going to take a little break for now.

**Comment by syndbg** [comment]
Updated

File: README.md [hunk 1]
```diff
@@ -144,7 +144,13 @@ Basic operations, you'll want to copy-paste this for testing purposes:
   window.plugins.calendar.deleteCalendar(calendarName,success,error);
 
   // create an event silently (on Android < 4 an interactive dialog is shown)
-  window.plugins.calendar.createEvent(title,eventLocation,notes,startDate,endDate,success,error);
+  window.plugins.calendar.createEvent(title,eventLocation,notes,startDate,endDate,{},success,error);
```
Comment (EddyVerbruggen) [inline README.md]: This `,{}` needs to be removed :)

**Comment by syndbg** [comment]
Had to edit with the online Github editor. Sorry about that.

**Comment by EddyVerbruggen** [comment]
It looks like `createEventInteractivelyWithOptions` is gone :(

Shall I go ahead and add it based on your PR?

**Comment by syndbg** [comment]
Yeah. Do so.

**Comment by EddyVerbruggen** [comment]
OK, thanks for your work!
It's part of 4.3.3.

**Pull request closed without merging**